### PR TITLE
Add documentation on command-line function to get client ID

### LINKED ISSUE #5546: composer/rest/get_client_id.py has has issues on giving arguments
## In which file did you encounter the issue?

composer/rest/get_client_id.py

### Did you change the file? If so, how?

Original:
```
    parser.add_argument('project_id', help='Your Project ID.')
    parser.add_argument(
        'location', help='Region of the Cloud Composer environent.')
    parser.add_argument(
        'composer_environment', help='Name of the Cloud Composer environent.')
```

Fixed locally:
```
    parser.add_argument('--project_id', help='Your Project ID.')
    parser.add_argument(
        '--location', help='Region of the Cloud Composer environent.')
    parser.add_argument(
        '--composer_environment', help='Name of the Cloud Composer environment.')
```

## Describe the issue

When running script error shows 

```
usage: id.py [-h] project_id location composer_environment
id.py: error: the following arguments are required: project_id, location, composer_environment
```

Needs to add '--' on the arguments. Issue was encountered here https://stackoverflow.com/questions/66400667

### PR DESCRIPTION
**Pull request by scouvreur**
## Description

Fixes #5546

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Review by dandhlee — CHANGES_REQUESTED** [review]
Thanks for adding this in! Just a minor comment below.

Otherwise, I think it might be worth putting in a note saying that this is an alternate in case there's issues with the `get_client_id.py` module, otherwise I was confused with why there's two of them until I read the related issue. Passing it to @leahecole for further review from here!

File: composer/rest/README.rst [hunk 1]
```diff
@@ -61,6 +61,15 @@ Install Dependencies
 Samples
 -------------------------------------------------------------------------------
 
+Determine client ID associated with a Cloud Composer environment
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+You can also get the client ID using the following command, where `AIRFLOW_URL`_ would be the URL of your Airflow webserver (ex. https://*****************-tp.appspot.com)
+
+.. code-block::bash
```
[the remark this answers is not in the document]
Comment (scouvreur) [reply composer/rest/README.rst:69]: Thanks @dandhlee - just addressed this in a recent commit

**Comment by scouvreur** [comment]
Hey @leahecole - sorry to disturb, any update on this ? Does this change look okay to you ?

**Review by leahecole — APPROVED** [review]
Thanks for submitting!

File: composer/rest/README.rst [hunk 2]
```diff
@@ -61,14 +61,37 @@ Install Dependencies
...
+
+    positional arguments:
+      project_id            Your Project ID.
+      location              Region of the Cloud Composer environment.
+      composer_environment  Name of the Cloud Composer environment.
+
+    optional arguments:
+      -h, --help            show this help message and exit
+
+In case of any issue with `get_client_id.py`_, you can also get the client ID using the following command, where `AIRFLOW_URL`_ would be the URL of your Airflow webserver (ex. `https://*****************-tp.appspot.com`_):
```
Comment (dandhlee) [inline composer/rest/README.rst:84]: nit: extra underscore ("_") after `get_client_id.py` and `AIRFLOW_URL`

Comment (dandhlee) [reply composer/rest/README.rst:84]: and `https://*****************-tp.appspot.com`

**Review by dandhlee — APPROVED** [review]

File: composer/rest/README.rst (hunk 2 above)
Comment (scouvreur) [reply composer/rest/README.rst:84]: No problem - thanks @dandhlee !

**Comment by scouvreur** [comment]
@dandhlee @leahecole thanks for your comments ! I addressed the changes, feel free to merge whenever you can as I don't have permissions 🙂

**Pull request merged**